### PR TITLE
Added blurple

### PR DESCRIPTION
**Pull request by GetRektByMe**

### CODE REVIEW

File: src/utils/colour.rs [hunk 1]
```diff
@@ -276,6 +276,8 @@ colour! {
     red, 0xE74C3C;
     /// Creates a new `Colour`, setting its RGB value to `(26, 188, 156)`.
     teal, 0x1ABC9C;
+    /// Creates a new `Colour`, setting its RGB value to `(114, 137, 218)`.
+    blurple, 0x7289da;
```
Comment (zeyla) [inline src/utils/colour.rs]: Capitalize the letters for consistency, please

Comment (zeyla) [inline src/utils/colour.rs]: Capitalize the letters for consistency, please

**Comment by abalabahaha** [comment]
hey don't steal my PR :c

**Comment by GetRektByMe** [comment]
Abal is a scrub

**Comment by zeyla** [comment]
Okay actually ignore what I said, I'll just merge as is

**Comment by zeyla** [comment]
Rebased in e9282d3373158b6e9792a5484ae3dfb9212eb6f.

**Pull request closed without merging**